docs/sources/source/nlp/bert_pretraining.rst

```diff
@@ -67,8 +67,8 @@ use the script ``examples/nlp/language_modeling/process_wiki_zh.py`` for preproc
 
 For already preprocessed data, we will be using a large dataset composed of Wikipedia and BookCorpus as in the original BERT paper.
 
-To download the dataset, go to `https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/LanguageModeling/BERT <https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/LanguageModeling/BERT>`__
-and run the script ``./data/create_datasets_from_start.sh``.
+To download the dataset, go to `https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/LanguageModeling/BERT <https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/LanguageModeling/BERT>`__,
+follow step 1-5 in the Quick-Start-Quide and run the script ``./data/create_datasets_from_start.sh`` inside the docker container.
```

Comment:
can you add the link to this guide please + typo in `Quide`

Reply:
ok